zap/src/main/java/org/parosproxy/paros/CommandLine.java

```diff
@@ -49,6 +49,7 @@
 // ZAP: 2020/11/26 Use Log4j 2 classes for logging.
 // ZAP: 2021/05/14 Remove redundant type arguments.
 // ZAP: 2022/02/09 No longer parse host/port and deprecate related code.
+// ZAP: 2022/02/28 Removed deprecated getConfigs()
```

Comment:
Could just say `Remove code deprecated in 2.6.0`, same in other files.

Reply:
This was marked as resolved but not code change or following up comment.

Reply:
oh I'm sorry I thought it was just a suggestion for future contribution because I took this template from Simon old pull req

Reply:
It is but I'd be better if done mainly to avoid the multiline comments.

Reply:
Done!